public/logTemplate.js

```diff
@@ -0,0 +1,80 @@
+function createLogTemplate(selectedPilot = null, isGM = false, isZen = false) {
```

Comment:
So instead of havig to define Zen or Gm, why don't you just set the selected Piot to those values instead.